lib/cob_index/indexer_config.rb

```diff
@@ -40,10 +40,8 @@
   provide "marc_source.type", "xml"
   # set this to be non-negative if threshold should be enforced
   provide "solr_writer.max_skipped", -1
-  # extend commit timeout
-  provide "solr_writer.commit_timeout", (15 * 60)
```

Comment:
This got added way back when we were still using traject commits in the Jenkins Job context:
tulibraries/tul_cob#427

So I think we're good to remove it because the default will work in the local dev context.